simglucose/analysis/report.py

```diff
@@ -93,16 +93,20 @@ def percent_stats(BG, ax=None):
 
 
 def risk_index_trace(df_BG, visualize=False):
-    chunk_BG = [df_BG.iloc[i:i + 60, :] for i in range(0, len(df_BG), 60)]
+    chunk_BG = [df_BG.iloc[i:i + 20, :] for i in range(0, len(df_BG), 20)]
```

Comment:
Why changing 60 to 20? Maybe you can make it an input parameter so people can customize it.

Reply:
My apologies. Now I understand you want to reflect the actual one hour since the sample time is 3 min. Yeah, your implementation is right, but actually different CGM sensor could have different sample time. It would be better we could get the sample time somehow.

Reply:
@jxx123 Thanks for the advice! Just added the parameter.